command/helpers/common.go

```diff
@@ -130,11 +130,21 @@ func FilterProperties(data []byte, filter []string) []byte {
 
 // ExecCommand Used to execute the remote calls for each of the managing commands
 func ExecCommand(basepath string, endpoint string, serverOverride string, get bool, Options string, Filter string, RawJSON bool) {
+    if DebugEnabled {
+        fmt.Fprintf(os.Stdout, "DEBUG [ExecCommand]: basepath->'%s', endpoint->'%s', serverOverride->'%s', get->'%t', Options->'%s', Filter->'%s', RawJSON->'%t'\n",
+            basepath, endpoint, serverOverride, get, Options, Filter, RawJSON)
+    }
     uri := GenerateURI(basepath, endpoint, serverOverride)
     response := RestCall(uri, get,  Options)
     if Filter == "" {
+        if DebugEnabled {
+            fmt.Fprintf(os.Stdout, "DEBUG [ExecCommand]: No Filter\n")
+        }
```

Comment:
A bit too match?

Reply:
hmm Probably is but it was to help when i was looking at @frenck stuff his trace made it look like he was in the Function when he shouldnt have been in teh end it was a red herring.

But I do have a statement that has it in the Function itself so yea i'll pull it.